apps/nextjs-example/pages/index.tsx

```diff
@@ -20,293 +21,385 @@ const WalletSelectorAntDesign = dynamic(
     wallet,
-    signAndSubmitTransaction,
-    signAndSubmitBCSTransaction,
-    signTransaction,
-    signMessage,
-    signMessageAndVerify,
   } = useWallet();
 
-  const { autoConnect, setAutoConnect } = useAutoConnect();
-  const { setSuccessAlertMessage, setSuccessAlertHash } = useAlert();
-
-  const onSignAndSubmitTransaction = async () => {
-    const payload: Types.TransactionPayload = {
-      type: "entry_function_payload",
-      function: "0x1::coin::transfer",
-      type_arguments: ["0x1::aptos_coin::AptosCoin"],
-      arguments: [account?.address, 1], // 1 is in Octas
-    };
-    const response = await signAndSubmitTransaction(payload);
-    try {
-      await aptosClient.waitForTransaction(response.hash);
-      setSuccessAlertHash(response.hash, network?.name);
-    } catch (error) {
-      console.error(error);
-    }
-  };
-
-  const onSignAndSubmitBCSTransaction = async () => {
-    const token = new TxnBuilderTypes.TypeTagStruct(
-      TxnBuilderTypes.StructTag.fromString("0x1::aptos_coin::AptosCoin")
-    );
-    const entryFunctionBCSPayload =
-      new TxnBuilderTypes.TransactionPayloadEntryFunction(
-        TxnBuilderTypes.EntryFunction.natural(
-          "0x1::coin",
-          "transfer",
-          [token],
-          [
-            BCS.bcsToBytes(
-              TxnBuilderTypes.AccountAddress.fromHex(account!.address)
-            ),
-            BCS.bcsSerializeUint64(2),
-          ]
-        )
-      );
-
-    const response = await signAndSubmitBCSTransaction(entryFunctionBCSPayload);
-    try {
-      await aptosClient.waitForTransaction(response.hash);
-      setSuccessAlertHash(response.hash, network?.name);
-    } catch (error) {
-      console.error(error);
-    }
-  };
-
-  const onSignTransaction = async () => {
-    const payload: Types.TransactionPayload = {
-      type: "entry_function_payload",
-      function: "0x1::coin::transfer",
-      type_arguments: ["0x1::aptos_coin::AptosCoin"],
-      arguments: [account?.address, 1], // 1 is in Octas
-    };
-    const response = await signTransaction(payload);
-    setSuccessAlertMessage(JSON.stringify(response));
-  };
-
-  const onSignMessage = async () => {
-    const payload = {
-      message: "Hello from Aptos Wallet Adapter",
-      nonce: "random_string",
-    };
-    const response = await signMessage(payload);
-    setSuccessAlertMessage(JSON.stringify(response));
-  };
-
-  const onSignMessageAndVerify = async () => {
-    const payload = {
-      message: "Hello from Aptos Wallet Adapter",
-      nonce: "random_string",
-    };
-    const response = await signMessageAndVerify(payload);
-    setSuccessAlertMessage(
-      JSON.stringify({ onSignMessageAndVerify: response })
-    );
-  };
-
   return (
     <div>
       <h1 className="flex justify-center mt-2 mb-4 text-4xl font-extrabold tracking-tight leading-none text-black">
-        Aptos Wallet Adapter Demo (Devnet)
+        Aptos Wallet Adapter Tester ({network?.name ?? ""})
```

Comment:
should we be worried for users accidentally submitting transactions to mainnet?

Reply:
None of these are a big deal, but I will make buttons disabled for Mainnet then.

Some wallets only support testnet, so I'm trying to keep it to allow at least connecting in all modes